Gopkg.lock

```diff
@@ -458,6 +458,6 @@
 [solve-meta]
   analyzer-name = "dep"
   analyzer-version = 1
-  inputs-digest = "e39a3b50eecf50ee2f3c6ce8a36306abeea762a41fab1117f0c5e2a038b72fb4"
+  inputs-digest = "d66983685b184f895ad58071cd04403f85b3b0d06b5818ca1c8225732f03b930"
```

Comment:
Is this needed if nothing in Gopkg.lock changed. Meaning it wasn't out of sync with Gopkg.toml.

Reply:
Here is a comment that I think might explain this conflict: https://github.com/golang/dep/issues/1224#issuecomment-333355751

I can not get this change to digest to not occur when running dep ensure. I think that means that anyone who takes this change, and If i manually revereted Gopkg.lock back to the old value, that when then ran `dep ensure` they would get a change to Gopkg.lock. I think we should include this change because of the that.